components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -100,6 +100,10 @@ spec:
           - name: TYK_PMP_PUMPS_HYBRID_META_CALLTIMEOUT
             value: "{{ .Values.pump.hybridPump.callTimeout }}"
             {{ end }}
+            {{ if .Values.pump.hybridPump.enableAggregateAnalytics }}
+          - name: TYK_MDCB_FORWARDANALYTICSTOPUMP
```

Comment:
This is MDCB environment variable. If pump is enabled(global.components.pump), then this should be set to true.